src/main/java/org/opentripplanner/model/transfer/TransferPoint.java

```diff
@@ -1,77 +1,119 @@
- *     position in pattern, not the stop for matching actual transfers. The reason is that
- *     real-time updates could invalidate a (route+stop)-transfer-point, since the stop could
- *     change to another platform(very common for railway stations). To account for this the
- *     RT-update would have to patch the (route&stop)-transfer-point. We simplify the RT-updates
- *     by converting expanding (route+stop) to (trip+stop position).
+ *     {@link StationTransferPoint} This apply to all trip stopping at a stop part of the given
+ *     station.
+ *     <p>The specificity-ranking is above {@link StationTransferPoint}s and less than
+ *     {@link RouteStationTransferPoint}.
+ *   </li>
+ *   <li>
+ *     A {@link RouteStationTransferPoint} is a from/to point for a Route at the given stop. This
+ *     only exist in GTFS, not in the Nordic NeTex profile.
  *
  *     <p>The specificity-ranking is above {@link StopTransferPoint}s and less than
+ *     {@link RouteStopTransferPoint}.
+ *   </li>
+ *   <li>
+ *     A {@link RouteStopTransferPoint} is a from/to point for a Route at the given station. This
+ *     only exist in GTFS, not in the Nordic NeTex profile.
+ *
+ *     <p>The specificity-ranking is above {@link RouteStationTransferPoint}s and less than
  *     {@link TripTransferPoint}.
  *   </li>
  *   <li>
  *     {@link TripTransferPoint} A transfer from/to a Trip at the given stop position(not stop).
- *     GTFS Transfers specify a transfer from/to a trip and stop. But in OTP we map the stop to a
- *     stop position in pattern instead. This make sure that the transfer is still valid after a
- *     real-time update where the stop is changed. Especially for train stations changing the
- *     train platform is common and by using the stop position in pattern not the stop this
- *     become more robust. So, the OTP implementation follow the NeTEx Interchange definition
- *     here, not the GTFS specification.
- *
- *     <p>This is the most specific point type, and will override both {@link RouteTransferPoint}
- *     and {@link StopTransferPoint} if more than one match exist.
+ *     The GTFS Transfers may specify a transfer from/to a trip and stop/station. But in OTP we
+ *     map the stop to a stop position in pattern. The OTP model {@link TripTransferPoint} do NOT
+ *     reference the stop/station, but the {@code stopPositionInPattern} instead. There is two
+ *     reasons for this. In NeTEx the an interchange is from a trip and stop-point, so this model
+ *     fits better with NeTEx. The second reson is that real-time updates could invalidate the
+ *     trip-transfer-point, since the stop could change to another platform(common for railway
+ *     stations). To account for this the RT-update would need to patch the trip-transfer-point.
+ *     We simplify the RT-updates by converting the stop to a stop-position-in-pattern.
+ *     <p>
+ *     This is the most specific point type.
```

Comment:
```suggestion
 *     The GTFS Transfers may specify a transfer from/to a trip and stop/station. But in OTP we
 *     map the stop to a stop position in pattern. The OTP model {@link TripTransferPoint} does NOT
 *     reference the stop/station, but the {@code stopPositionInPattern} instead. There is two
 *     reasons for this. In NeTEx the an interchange is from a trip and stop-point, so this model
 *     fits better with NeTEx. The second reason is that real-time updates could invalidate the
 *     trip-transfer-point, since the stop could change to another platform(common for railway
 *     stations). To account for this the RT-update would need to patch the trip-transfer-point.
 *     We simplify the RT-updates by converting the stop to a stop-position-in-pattern.
 *     <p>
 *     This is the most specific point type.
```

Reply:
Would it make sense to make the Netex-based transfer to reference the Station, which would solve this issue?